SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005822	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio2}#%{nl}Install count: #{CaptionRatio}#
+SKILL_20151224_005823	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionTime}#%{nl}Arrow count: #{CaptionRatio}#{nl}Install range: #{CaptionRatio2}#{nl}Consume 2 wires
+SKILL_20151224_005824	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}The companion bites and hold the enemy in place. The bitten enemy will have their physical defense reduced while critical hit rate bonus is applied against the enemy.
+SKILL_20151224_005825	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}The companion bites an airborne enemy and lower them to the ground.
+SKILL_20151224_005826	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}The companion bites and drags the enemy towards their owner.
+SKILL_20151224_005827	Awareness of hidden enemy count: #{CaptionRatio}#{nl}Hounding AI duration: 90 second
+SKILL_20151224_005828	Cure a poisoned comrade. If the poison is higher rank than the antidote, duration of the poison is reduced instead.
+SKILL_20151224_005829	{#DD5500}{ol}[Physical] - [Poison]{/}{/}{nl}Create a poison puddle by throwing a pot filled with poison. Any enemies who come into contact with the puddle is poisoned.
+SKILL_20151224_005830	{#DD5500}{ol}[Physical] - [Poison]{/}{/}{nl}Throw Jincan to attack enemies. The defeated enemies is replaced with newly born poison insect.
+SKILL_20151224_005831	Block count: #{CaptionRatio}#{nl}Moving speed reduction: #{CaptionRatio2}#
+SKILL_20151224_005832	Skill critical hit rate increase: #{CaptionRatio}#%
+SKILL_20151224_005833	Wild Shot
+SKILL_20151224_005834	{#DD5500}{ol}[Missile]{/}{/}{nl}Attack simultaneously with crossbow and pistol held in both hands. 
+SKILL_20151224_005835	Sonic Strike
+SKILL_20151224_005836	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Quickly descend the hawk to attack the enemies in the designated area.
+SKILL_20151224_005837	{#DD5500}{ol}[Missile]{/}{/}{nl}Fire cannonballs to target the enemies on ground. The damage is halved against airborne enemies.
+SKILL_20151224_005838	{#DD5500}{ol}[Missile]{/}{/}{nl}Fire cannonballs to target airborne enemies. The damage is halved against enemies on ground.
+SKILL_20151224_005839	{#DD5500}{ol}[Missile]{/}{/}{nl}Fire a howitzer to the ground. 5x damage is dealt to enemy's structure.
+SKILL_20151224_005840	{#DD5500}{ol}[Missile]{/}{/}{nl}Fire a cannonball that spread to target all enemies in front of you. The user can be pushed back from the recoil of the shot.
+SKILL_20151224_005841	Attack: 150% + #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%
+SKILL_20151224_005842	{#DD5500}{ol}[Missile]{/}{/}{nl}Attack multiple enemies by firing at a designated area.
+SKILL_20151224_005843	{#DD5500}{ol}[Missile]{/}{/}{nl}Accurately aim at the head of the enemy to attack. The attacked enemy will have their intelligence and spirit temporality reduced.
```

Comment:
Aim accurately at the head of an enemy to attack. The attacked enemy will have its INT and SPR temporarily reduced.